psf_proposal.md

```diff
@@ -0,0 +1,88 @@
+# Proposal for Python Software Foundation
+
+## Form
+
+- Contact
+
+  - Legal First Name*: Ivan
+
+  - Legal Last Name *: Ogasawara
+
+  - Nickname:
+
+  - Email *: ivan.ogasawara@gmail.com
+
+- Address (selection)
+
+  - City *: La Paz
+
+  - Country*: Bolivia
+
+  - State/Province/Region: La Paz
+
+  - Region Other:
+
+  - Postal Code:
+
+  - Postal Code Suffix:
+
+  - Phone Number *:
+
+- Organization
+
+Company or organizing organization, example: Chicago PyLadies
+  
+  - Organization Name: Open Science Labs
+
+  - Organization Email: opensciencelabs@gmail.com
+
+  - Twitter Handle: @opensciencelabs/
+
+  - Website: https://opensciencelabs.org/
+
+  - Grant Type *: Development work
+
+  - Grant Title *: OSL Cookiecutter
+  Example: The fifth annual PyCon Atlantis
+
+- FAQ & Grants Program Info *:
+
+    I have read and understand these pages (es una casilla de selección)
+    Note: Before completing this application, read the web pages pertaining to the PSF’s grants program, freely available tools to run virtual meetings, and understand how funding works:
+    https://www.python.org/psf/grants
+    https://www.python.org/psf/grants/free-resources/
+
+Date *: 
+MM/DD/YYYY
+Provide the start date for your event/project
+Event’s/project’s website *:
+
+Requested Grant Amount (USD) *: 
+
+Grant Amount Breakdown *:
+
+What amount are you requesting from the PSF? If the grant is funding several components of a project/event, please provide a breakdown of what the funding will be used for. Example: Our event is requesting $1500 total ($1000 for captioning/transcripts, $500 for improved internet bandwidth for speakers).
```

Comment:
I need to think a bit about that .. but I guess that almost all or all of the bugdet would be for development